sdk/client/src/otel/index.ts

```diff
@@ -285,7 +285,7 @@ const getSpanName = (
 	let parsedBody
 	const urlObject = new URL(url)
 	const pathname = urlObject.pathname
-	let spanName = `${method} - ${pathname}`
+	let spanName = `${method.toUpperCase()} - ${pathname}`
```

Comment:
@ccschmitz this change is unrelated, right?

Reply:
Oh yeah, sorry... Can back this change out probably

Reply:
all good, can also have this here but i'll add a changelog to release the client